### PR TITLE
added normalize.css package as reset css

### PR DESCRIPTION
**Pull request by akiskips**
## 🛠️ Description
Added normalize.css package from https://github.com/necolas/normalize.css/
Added import at main.tsx

Fixes # 

### 🔬 How did you test this change?

> - [ ] 🤏 Lint tests
> - [ ] 🤞 PS -WhatIf / az validate
> - [x] 👍 Manually deployed + verified
> - [ ] 💪 Unit tests
> - [ ] 🙌 Integration tests

### 🙋‍♀️ Do any of the following that apply?

> - [ ] 🚨 This is a breaking change.
> - [ ] 🤏 The change is less than 20 lines of code.

### 📑 Did you update `docs/changelog.md`?

> - [ ] ✅ Updated changelog (required for `dev` PRs)
> - [ ] ➡️ Will add log in a future PR (feature branch PRs only)
> - [x] ❎ Log not needed (small/internal change)

### 📖 Did you update documentation?

> - [ ] ✅ Public docs in `docs` (required for `dev`)
> - [ ] ✅ Internal dev docs in `src` (required for `dev`)
> - [ ] ➡️ Will add docs in a future PR (feature branch PRs only)
> - [x] ❎ Docs not needed (small/internal change)

### CODE REVIEW

**Comment by akiskips** [comment]
@Ivanmtta Not sure if it would be better to add an @import to index.css instead

**Review by flanakin — APPROVED** [review]

File: src/web/package.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
   },
   "dependencies": {
     "eslint-plugin-react": "^7.37.1",
+    "normalize.css": "^8.0.1",
```
Comment (Ivanmtta) [inline src/web/package.json:15]: Not sure if this is necessary. Cold we just add this to the main css file?
```
* {
  padding: 0;
  margin: 0;
  box-sizing: border-box;
}
```

Comment (akiskips) [reply src/web/package.json]: Sure we can do that . I removed the package and added the reset rule in the start of index.css
My idea was to use the normalize.css that seems to add a variaty of reset rules for different browsers. But it adds to complexity and probably we dont need it atm:

https://github.com/necolas/normalize.css/blob/master/normalize.css

Comment (flanakin) [reply src/web/package.json]: Ivan and I discussed this. We agreed to revert back to normalize.css out of the box.

@akiskips You can resolve this comment when you've done that. Thanks!

Comment (akiskips) [reply src/web/package.json:15]: Reverted to the normalize.css package.

**Review by flanakin — APPROVED** [review]

**Pull request merged**